spec/system/admin/admin_manages_pages_spec.rb

```diff
@@ -124,15 +124,15 @@
 
       it "allows a landing page to be updated", :aggregate_failures do
         visit edit_admin_page_path(current_landing_page.id)
-        expect(page).to have_content("Use as 'Locked Screen")
+        expect(page).to have_content("Use as 'Locked Screen'")
```

Comment:
This and all following, related changes (lines `135`, `145`, and `158`) are merely nitpicks that add a closing quote to match the text on the actual modal.